week2/homework/src/doList.json

```diff
@@ -0,0 +1,5 @@
+{
+  "1": "Do your homework buddy!",
+  "2": "Practise to master it.",
+  "3": "coding with love my friend"
+}
```

Comment:
This file is not used, so you need to replace it with the file you are using ( `todoList.txt`)

Reply:
Hey, I didn't notice that when i pushed my homework. This file is of my old homework so i think they mixed up with the same old branch..